composer.json

```diff
@@ -56,5 +62,11 @@
         "branch-alias": {
             "dev-master": "2.x-dev"
         }
-    }
+    },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "git@github.com:acrobat/Testing.git"
```

Comment:
@acrobat travis have mostly problems with forks as you see the CI fails to clone the dependency. You can work around this problems by adding the following to this composer.json:

```json
"config" {
    "github-protocols": ["https"]
}
```

So we can see if the tests run successfully.

Reply:
i merged https://github.com/symfony-cmf/Testing/pull/202 now, using latest master of the original testing repo should pick that up now